css/properties/display.json

```diff
@@ -1089,14 +1089,40 @@
               "edge_mobile": {
                 "version_added": false
               },
-              "firefox": {
-                "version_added": true,
-                "notes": "Scheduled for removal (See <a href='https://bugzil.la/879275'>bug 879275</a>)."
-              },
-              "firefox_android": {
-                "version_added": true,
-                "notes": "Scheduled for removal (See <a href='https://bugzil.la/879275'>bug 879275</a>)."
-              },
+              "firefox": [
+                {
+                  "version_added": true,
+                  "version_removed": "64",
+                  "notes": "This is still available to Firefox UI code"
```

Comment:
I would change these notes to "Still available in Firefox UI code."
Notes should end with a full stop. (we should probably document this requirement)